core/src/nusi/fixed-select.tsx

```diff
@@ -17,9 +17,13 @@ import { get } from 'lodash';
 
 const { Option, OptGroup } = Select;
 const FixedSelect = React.forwardRef((props, ref) => {
-  const options = props.children || get(props, 'options');
+  const { options: propsOptions, ...rest } = props;
+  
+  //Determine if the child elements of props.options is components
+  const isComponent = props.options?.find(item => item.type?.name === 'Option');
+  const options = isComponent ? props.children || get(props, 'options') : '';
```

Comment:
if user pass `props.children`, will u still need to check whether its React component?

Reply:
I think it's not necessary, when users pass props.children, it should be placed in <Select></Select>, users should not be passing Object or Array and so on.